### ISSUE TITLE
Discourage use of informalfigure

### ISSUE THREAD

**Issue by janajaeger** [SUSE/doc-styleguide#311]
Here is a candidate for our future accessibility section:

All images in the docs need an alternative text that can be processed by screenreaders. Currently, we allow the use of informalfigure without any caption at all which results on a ton of missing alt texts and heavy penalties on Siteimprove. To achieve compliance with the upcoming EU regulations (in June 2025), we need to adress this.

In the short to midterm, Toms will invent some stylesheet magic that generates an alt text. We need this to avoid having to update all maintenance branches and languages.

In the long term, we should just discourage writers from using this element at all.

@dariavladykina feel free to add any missing info that you and @tomschr came up with. Thanks a million.

**Comment by tomschr** [comment]
As discussed on Slack (and liked by Frank), this would be the algorithm:

1. Look for a `<textobject>`in the `informalfigure`:
   1. If found, use it and stop processing.
   1. If it doesn't exist, continue.
1. Extract the title from the hierarchy element (section, chapter, ...).
1. If we have more than one `<informalfigure>`, number it.
1. Combine the title from step 2 and the number from step 3.
1. Done.

**Comment by dariavladykina** [comment]
I'd suggest that I go through the main branches + latest maintenance branches in our repos and correct existing issues. As we have calculated with Toms today, there are not many in 'Linux' docs.
As of next releases, let's instruct the writers to not use <informalfigure> and stick to <figure>.

**Comment by dariavladykina** [comment]
[Informalfigure and textobjectrole search results.odt](https://github.com/user-attachments/files/18556320/Informalfigure.and.textobjectrole.search.results.odt)

**Comment by dariavladykina** [comment]
Implemented, closing.
